packages/server-wallet/src/utilities/messaging.ts

```diff
@@ -44,8 +44,8 @@ function mergeProp<T>(a: T[] | undefined, b: T[] | undefined): T[] | undefined {
   if (a && b) {
     return _.orderBy(
       _.uniqWith(_.concat(a, b), _.isEqual),
-      ['channelId', 'turnNum'],
-      ['desc', 'asc']
+      ['channelNonce', 'turnNum'],
+      ['asc', 'asc']
```

Comment:
Flagging as a possible separate PR. To discuss: what order should `channelResults` and `signedStates` be in? I dislike using alphanumerically sorted `channelId`s since they are harder to predict.